package.json

```diff
@@ -69,15 +69,15 @@
     "test:e2e-nightly-prod": "MOCHAWESOME_REPORTFILENAME=e2e-nightly-prod FUNCTIONAL=true npx codeceptjs run-workers --suites 15 --grep '@e2e-nightly-prod' --reporter mocha-multi --verbose",
     "test:api-nightly-prod": "MOCHAWESOME_REPORTFILENAME=api-nightly-prod FUNCTIONAL=true npx codeceptjs run-workers --suites 8 --grep '@api-nightly-prod|@api-prod' --reporter mocha-multi --verbose",
     "test:e2e-nightly-nonprod": "MOCHAWESOME_REPORTFILENAME=non-prod-e2e-ft npx codeceptjs run-workers --suites 2 --grep '@e2e-nightly-nonprod' --reporter mocha-multi --verbose",
-    "test:non-prod-e2e-ft": "MOCHAWESOME_REPORTFILENAME=non-prod-e2e-ft FUNCTIONAL=true npx codeceptjs run-workers --suites 12 --grep '@master-e2e-ft|@non-prod-e2e-ft' --reporter mocha-multi --verbose",
+    "test:non-prod-e2e-ft": "FUNCTIONAL=true node run-tests.js non-prod-e2e-ft '@master-e2e-ft|@non-prod-e2e-ft'",
```

Comment:
If I remember correctly we were keeping this  tag "MOCHAWESOME_REPORTFILENAME" to make it compatible to run locally with both windows and mac both?